test/merge/config-absent/merge-config-absent.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+const { join } = require('path');
+
+const { run } = require('../../utils/test-utils');
+
+describe('merge flag configuration', () => {
+    it('Show warning message when the merge config is absent', () => {
+        // 2.js doesn't exist, let's try merging with it
+        const { stdout, stderr } = run(__dirname, ['--config', './1.js', '--merge', './2.js'], false);
+
+        // Since the process will exit, nothing on stdout
+        expect(stdout).toBeFalsy();
+        // Confirm that the user is notified
+        expect(stderr).toContain(`MergeError: The supplied merge config doesn't exist.`);
+        // Default config would be used
+        expect(fs.existsSync(join(__dirname, './dist/merged.js'))).toBeFalsy();
+        // Since the process will exit so no compilation will be done
+        expect(fs.existsSync(join(__dirname, './dist/main.js'))).toBeFalsy();
```

Comment:
Can we add test there we have the same options in both configuration, to ensure we override it from the second configuration?

Reply:
Added 👍

Reply:
Can't find this test :confused: What is overridden?

Reply:
Added in this commit - https://github.com/webpack/webpack-cli/pull/1674/commits/9b34377c562e60eeb6e59d7d4a6500fefb234746